scripts/build.js

```diff
@@ -0,0 +1,240 @@
+"use strict";
+
+const fs = require("fs");
+const path = require("path");
+const xmlb = require("xmlbuilder2");
+const SVGO = require("svgo");
+
+const svgoOptions = {
```

Comment:
Two questions:

1) How can we make sure this list will remain in sync with `svgo.yml`? Perhaps a comment on both locations would help in that regard
2) It's a valid approach to document all options explicitly, rather than only those set differently than svgo's defaults. However, that loses us that information, which I feel is valuable to illustrate what this project has deliberately chosen to configure. Maybe add comments next to the non-default lines specifying that they differ from the defaults?

Reply:
SVGO just got a new maintainer who's been doing some intensive work recently, and v2 was just released. It includes a [change](https://github.com/svg/svgo/issues/1327) that could help us out here—we could just import the config inside the script and avoid repetition. The YAML format is deprecated in the new version, though, replaced with a `svgo.config.js` exporting a config object.

The new version also adds `extendDefaultPlugins` as part of the new config scheme, which addresses the 2nd point too! No need for the long list of plugins.

This will require a bump to SVGO v2, I'll see if it doesn't break anything.

Reply:
Amazing news! Thanks for sharing. I would expect there not to be much breakage, and whatever differences may exist to be fixed via configuration. Of course, if it does turn out to be cumbersome to upgrade, we could certainly leave that as a future improvement for another PR.